ExtendedDescription.md

```diff
@@ -0,0 +1,130 @@
+
+Additionally, community organizers are available to help community members engage with local law enforcement or to otherwise help those experiencing unacceptable behavior feel safe.
+
+### Goals
+
+1. Provide a safe and welcoming community space for discussing Nelson & area activities, news, events, and experiences.
+1. Provide a venue for both casual and serious conversation, with the ability for users to opt-out of unwanted content.
+1. Fun! We aim to foster a community of individuals and business that love being here in the Kootenays. We welcome personal posts, event notices and organizing.
+
+### Non-Goals
+
+While it's important to know what nelson.social is, it's just as important to understand what nelson.social is not. Nelson.social is not:
+
+* A money-making exercise. This is a community-run website, made of open-source software, hosted and moderated by volunteers who live in your community. We are not going to [sell your data](https://en.wikipedia.org/wiki/Facebook%E2%80%93Cambridge_Analytica_data_scandal), or try to get you [addicted](https://www.theguardian.com/technology/2017/nov/09/facebook-sean-parker-vulnerability-brain-psychology) so that we can show you more ads. In fact, there are no ads!
+* A "free speech" zone. We encourage authentic, curious, respectful conversation, including disagreement, but we do not tolerate all forms of speech. We do not tolerate hate speech of any kind. [Bad-faith](https://en.wikipedia.org/wiki/Bad_faith) debating is equally unwelcome, as is misinformation.
+* A place for general advertisement. Promoting your original work is welcome, but please avoid overly commercial self-promotion
+
+You may [find another server](https://instances.social/) to be a better fit if you don't like these rules.
+
+### Encouraged Behavior
+
+The following behaviors are expected and requested of all community members:
+
+* Be friendly, open and welcoming.
+* Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
+* Exercise consideration and respect in your speech and actions.
+* Attempt collaboration & de-escalation before conflict.
+* If you inadvertently cause offence to someone else, take responsibility for your actions.
+* Refrain from demeaning, discriminatory, or harassing behavior and speech.
+* Be mindful of your fellow participants. Alert community leaders if you notice a dangerous situation, someone in distress, or violations of this Code of Conduct, even if they seem inconsequential.
+  
+### Unacceptable Behavior
+
+To be specific, the following behaviors are unacceptable within our community:
+
+* Anything in violation of our [Terms of Service](https://nelson.social/terms) including posting anything illegal in Canada.
```

Comment:
I just spend a week doing legal. We should say Canada or British Columbia, since that's where to court would take place.

Reply:
```suggestion
* Anything in violation of our [Terms of Service](https://nelson.social/terms) including posting anything illegal in British Columbia or Canada.
```